core/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -122,7 +122,7 @@ private Setting(Key key, @Nullable Setting<T> fallbackSetting, Function<Settings
         this.defaultValue = defaultValue;
         this.parser = parser;
         if (properties == null) {
-            throw new IllegalArgumentException("properties can not be null for setting [" + key + "]");
+            throw new IllegalArgumentException("properties cannot be null for setting [" + key + "]");
```

Comment:
I don't mind reverting it, but I've always assumed non-native English speakers have a hard time with this one. Personally, I was taught to _always_ use the former, but English being what it is leads to multiple interpretations of "rules".

For what it's worth: http://www.oxforddictionaries.com/words/cannot-or-can-not